src/autocomplete/databases/clickhouse/clickhouse-autocomplete.ts

```diff
@@ -158,13 +159,14 @@ function generateSuggestionsFromRules(
     let suggestViewsOrTables: ClickHouseAutocompleteResult['suggestViewsOrTables'];
     let suggestAggregateFunctions = false;
     let suggestFunctions = false;
+    let suggestDatabases = false;
     let shouldSuggestColumns = false;
     let shouldSuggestColumnAliases = false;
     let suggestEngines;
 
     for (const [ruleId, rule] of rules) {
         if (!isStartingToWriteRule(cursorTokenIndex, rule)) {
-            break;
+            continue;
```

Comment:
This is huge. My mistake.
We don't want to stop suggesting, we just want to skip a rule if we're not on it's beginning